engine/src/main/java/org/terasology/utilities/procedural/PerlinNoise.java

```diff
@@ -20,8 +20,10 @@
 
 /**
  * Improved Perlin noise based on the reference implementation by Ken Perlin.
+ * @deprecated Prefer using SimplexNoise, it is comparable to Perlin noise (fewer directional artifacts, lower computational overhead for higher dimensions).
```

Comment:
I think you can put a link to the class. Not sure about the syntax how to achieve this in JavaDoc, though...

Reply:
Added link